lib/archiving.js

```diff
@@ -49,6 +49,17 @@ var request = require('request'),
 *          <a href="https://dashboard.tokbox.com">OpenTok dashboard</a>.
 *   </ul>
 *
+* @param {Boolean} hasAudio
+*   The archive has an audio track. If hasAudio is set to true when calling OpenTok.startArchive,
```

Comment:
Change "OpenTok.startArchive" to: {@link OpenTok#startArchive OpenTok.startArchive()}

Also, we should not document these as parameters of the constructor. They should be documented as properties of the Archive object. (This goes for the existing @param declarations in the docs.) And the documentation should say that the developer should not call this constructor directly. (They should call OpenTok.startArchive().)